README.md

```diff
@@ -146,6 +147,25 @@ Then install the package:
 sudo zypper --no-gpg-checks install R-${R_VERSION}-1-1.x86_64.rpm
 ```
 
+Download the deb package:
+```bash
+# Debian 10
+curl -O https://cdn.posit.co/r/fedora-37/pkgs/r-${R_VERSION}_1_amd64.rpm
+
+# Debian 11
+curl -O https://cdn.posit.co/r/fedora-38/pkgs/r-${R_VERSION}_1_amd64.rpm
+
+# Debian 12
+curl -O https://cdn.posit.co/r/debian-12/pkgs/r-${R_VERSION}_1_amd64.deb
+```
```

Comment:
We'll need a new header for Fedora, and then the references to deb/Debian should be removed here

Reply:
This is fixed by afa8fdd